src/settings/components/LinkedAccounts.tsx

```diff
@@ -205,10 +212,8 @@ const LinkedAccounts: FunctionComponent<AccountProps> = ({ location, user }) =>
 										'settings/components/account___koppel-je-account-met-andere-platformen'
 									)}
 								>
-									{/* Disabled for pupils due to https://meemoo.atlassian.net/browse/AVO-2062 */}
-									<div>
-										{!isPupil && renderIdpLinkControls('VLAAMSEOVERHEID')}
-									</div>
+									{/* Previously disabled for pupils due to https://meemoo.atlassian.net/browse/AVO-2062 */}
+									<div>{renderIdpLinkControls('VLAAMSEOVERHEID')}</div>
```

Comment:
button is not the same size on mobile:
![image](https://user-images.githubusercontent.com/1710840/188839160-a0dff26a-778d-458a-8879-d4869cfed4dc.png)